4] guessing game 3/main.py

```diff
@@ -0,0 +1,51 @@
+from flask import Flask, request, render_template
+
+app = Flask(__name__)
+
+min_num = 0
+max_num = 1000
+guess = 0
+
+def calculation(minimum=int, maximum=int) -> int:
+    return int((maximum-minimum) / 2) + minimum
+
+
+@app.route("/too_low", methods=["GET", "POST"])
+def too_low():
+    global min_num, max_num, guess
```

Comment:
uff .. gobalni promenne jsou dost bad practise, to se moc neuc. To se dela opravdu jen v pripadech kde se musi neco hodne ohnout.

Reply:
Chápu, je mi jasný, že je to řešení poněkud na čuňátko... ale já prostě nemohla přijít na to, jak jinak to udělat. Neměl bys prosím nějakou radu?

Reply:
no ja myslim ze mas hlavne udelat jen jedno tu routu `@app.route("/", methods=["GET", "POST"])`
 a v ni mit celou tu logiku, ktera je v podstate stejna jako v tom predeslem ukolu, jenom je to obalene do HTML.